x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_indices/indices_logic.ts

```diff
@@ -92,19 +107,29 @@ export const IndicesLogic = kea<MakeLogicType<IndicesValues, IndicesActions>>({
   },
   connect: {
     actions: [
+      CancelSyncsApiLogic,
+      ['apiSuccess as cancelSuccess'],
+      FetchIndexApiLogic,
+      ['makeRequest as fetchIndexDetails'],
       FetchIndicesAPILogic,
       ['makeRequest', 'apiSuccess', 'apiError'],
       DeleteIndexApiLogic,
       ['apiError as deleteError', 'apiSuccess as deleteSuccess', 'makeRequest as deleteIndex'],
     ],
     values: [
+      FetchIndexApiLogic,
+      ['data as indexDetails', 'status as indexDetailsStatus'],
       FetchIndicesAPILogic,
       ['data', 'status'],
       DeleteIndexApiLogic,
       ['status as deleteStatus'],
     ],
   },
   listeners: ({ actions, values }) => ({
+    cancelSuccess: async (_, breakpoint) => {
+      await breakpoint(500); // give connectors a chance to stop before checking again
```

Comment:
non-blocking nit: would be nice to use a shared constant for this `500`. 1 to not have a magic number but 2. to have predefined timeouts to use for breakpoints in other logics.

Like why is 500ms right here but we use 150ms on line 138 for `fetchIndices` :)

Reply:
This is a very good point, but actually we don't need this timeout here so I'm just removing it instead <_<